### ISSUE TITLE
Profile does not change after login

### ISSUE THREAD

**Issue by yathit** [swanhtetkyaw/Fertilizer-app-vue#2]
Thanks for sharing very good looking web site. I tried adding sale and search and working fine.

After login with my account, the profile does not reflect to my account.

<img width="998" alt="Screenshot 2020-06-24 at 10 05 54 AM" src="https://user-images.githubusercontent.com/2137932/85489655-e6683400-b602-11ea-8caa-bc99f57cdc67.png">

**Comment by swanhtetkyaw** [comment]
Yes,because I have some problem with that i will talk you in depth later .I
am now taking a bath
